src/Entity/Realm.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of letswifi; a system for easy eduroam device enrollment
+ * Copyright: 2023, Paul Dekkers, SURF <paul.dekkers@surf.nl>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+namespace App\Entity;
+
+use App\Repository\RealmRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: RealmRepository::class)]
+class Realm
+{
+    #[ORM\Id]
+    #[ORM\Column(length: 127)]
+    private string|null $realm = null;
+
+    /** @var Collection<RealmContact> */
+    #[ORM\OneToMany(mappedBy: 'contact', targetEntity: RealmContact::class)]
```

Comment:
Zou deze niet mappedBy `realm` moeten zijn?

Reply:
Klopt, met bovenstaande is daarmee de laatste schema:validation gefixed